src/sentry/dynamic_sampling/rules/combine.py

```diff
@@ -0,0 +1,33 @@
+from sentry.dynamic_sampling.rules.biases.boost_environments_bias import BoostEnvironmentsBias
+from sentry.dynamic_sampling.rules.biases.boost_key_transactions_bias import (
+    BoostKeyTransactionsBias,
+)
+from sentry.dynamic_sampling.rules.biases.boost_latest_releases_bias import BoostLatestReleasesBias
+from sentry.dynamic_sampling.rules.biases.ignore_health_checks_bias import IgnoreHealthChecksBias
+from sentry.dynamic_sampling.rules.biases.uniform_bias import UniformBias
+from sentry.dynamic_sampling.rules.combinators.base import BiasesCombinator
+from sentry.dynamic_sampling.rules.combinators.ordered_combinator import OrderedBiasesCombinator
+from sentry.dynamic_sampling.rules.utils import RuleType
+
+
+def _get_default_combinator() -> BiasesCombinator:
+    # The default combinator is the ordered combinator, which will keep the insertion order of the rules.
+    return OrderedBiasesCombinator()
+
+
+def get_relay_biases_combinator() -> BiasesCombinator:
+    default_combinator = _get_default_combinator()
```

Comment:
i don't understand why do we need method to return an instance of the class? why not just replace with `default_combinator = OrderedBiasesCombinator()`

Reply:
The reasoning was that I wanted to explicitly isolate the combinator definition but we can for sure inline it.